tools/postinstall-fixup/linux-postinstall-fixup.sh

```diff
@@ -48,21 +48,34 @@ rm -rf "$INSTALL_DIR/plugins/platforms"
 rm -rf "$INSTALL_DIR/plugins/styles"
 rm -rf "$INSTALL_DIR/plugins/xcbglintegrations"
 
+# Qt plugins
+QT_PLUGIN_DIR="$QT_DIR/plugins"
+
+# Qt installed via Fedora RPM
+if ! [ -d "$QT_PLUGIN_DIR" ] && [ -d "/usr/lib64/qt5/plugins" ] ; then
+  QT_PLUGIN_DIR="/usr/lib64/qt5/plugins"
+fi
+
+# Qt installed via Ubuntu DEB
+if ! [ -d "$QT_PLUGIN_DIR" ] && [ -d "/usr/lib/x86_64-linux-gnu/qt5/plugins" ] ; then
+  QT_PLUGIN_DIR="/usr/lib/x86_64-linux-gnu/qt5/plugins"
+fi
+
```

Comment:
What is the (I guess wrong) value of QT_DIR with Fedora RPM and Ubuntu DEB?
I think it would be better to fix QT_DIR instead of adding hardcoded guesses.

Reply:
> What is the (I guess wrong) value of QT_DIR with Fedora RPM and Ubuntu DEB?
> I think it would be better to fix QT_DIR instead of adding hardcoded guesses.

There isn't:

```
/usr/lib64/libQt5Core.so
/usr/lib/x86_64-linux-gnu/libQt5Core.so
```

Reply:
I don't understand. My question was unclear I think.
What do you get if you add `message("Qt5_DIR = ${Qt5_DIR}")` at the end of SOFA main CMakeLists?